main.go

```diff
@@ -38,6 +38,7 @@ var (
 	showVersion         = flag.Bool("version", false, "Print version information.")
 	listenAddress       = flag.String("web.listen-address", ":9091", "Address to listen on for the web interface, API, and telemetry.")
 	metricsPath         = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
+	prefixPath          = flag.String("web.external-url", "", "The URL under which PushGateway is externally reachable.")
```

Comment:
Sorry, that's two lines further down: https://github.com/prometheus/prometheus/blob/master/cmd/prometheus/config.go#L303